apps/src/studio/skins.js

```diff
@@ -417,15 +417,24 @@ function loadHoc2015(skin, assetUrl) {
      [0x00, 0x00,  0x00,  0x00,  0x00, 0x00,  0x121, 0x121]];
 
   // Sounds.
-  skin.character1sound1 = [skin.assetUrl('character1sound1.mp3'), skin.assetUrl('wall.ogg')];
-  skin.character1sound2 = [skin.assetUrl('character1sound2.mp3'), skin.assetUrl('wall.ogg')];
-  skin.character1sound3 = [skin.assetUrl('character1sound3.mp3'), skin.assetUrl('wall.ogg')];
-  skin.character1sound4 = [skin.assetUrl('character1sound4.mp3'), skin.assetUrl('wall.ogg')];
+  var sounds = [
+    'character1sound1', 'character1sound2', 'character1sound3', 'character1sound4',
+    'character2sound1', 'character2sound2', 'character2sound3', 'character2sound4',
+    'item1sound1', 'item1sound2', 'item1sound3', 'item1sound4',
+    'item3sound1', 'item3sound2', 'item3sound3', 'item3sound4',
+    'alert1', 'alert2', 'alert3', 'alert4',
+    'applause',
+    'start', 'win', 'failure', 'flag'
+  ];
 
-  studioApp.loadAudio(skin.character1sound1, 'character1sound1');
-  studioApp.loadAudio(skin.character1sound2, 'character1sound2');
-  studioApp.loadAudio(skin.character1sound3, 'character1sound3');
-  studioApp.loadAudio(skin.character1sound4, 'character1sound4');
+  // Override the default loadAudio function with this one.
+  skin.loadAudio = function() {
+    for (var s = 0; s < sounds.length; s++) {
+      var sound = sounds[s];
+      skin[sound] = [skin.assetUrl(sound + '.mp3'), skin.assetUrl('wall.ogg')];
+      studioApp.loadAudio(skin[sound], sound);
+    }
```

Comment:
The way you have it is perfectly acceptable. Note another way you could do this is
```
sounds.forEach(function (sound) {
  skin[sound] = ..;
  studioApp.loadAudio(...)
});
```
If we were writing ES6, you could just do
```
for (var s of sounds) {
  skin[sound] = ..;
  studioApp.loadAudio(...)
}
```
But I digress. How you have it is fine :)